lib/wallet.py

```diff
@@ -1170,7 +1170,7 @@ def get_tx_status(self, tx_hash, height, conf, timestamp):
             if fee is not None:
                 size = tx.estimated_size()
                 fee_per_byte = fee / size
-                extra.append('%.1f sat/b'%(fee_per_byte))
+                extra.append(format_satoshis(fee_per_byte, precision=FEE_PRECISION) + ' sat/b')
```

Comment:
this won't work
in `format_satoshis` you have
```
result = ("{:" + decimal_format + "f}").format(x / pow (10, decimal_point)).rstrip('0')
```
which divides `fee_per_byte` by 10**8.
So at the very least you should set `decimal_point` to `0` here.
I guess this is a sign that `format_satoshis` is a bit confusing. :/

Reply:
Yeah that's interesting it's something of a formatting and conversion function in one - sat -> BTC -> mBTC on the one hand, zeroes / padding / precision on the other. Maybe that's the line to divide it along.